Symfony/CS/Tests/Fixer/PSR2/MethodArgumentSpaceFixerTest.php

```diff
@@ -29,6 +29,20 @@ public function testFix($expected, $input = null)
     public function testFixProvider()
     {
         return array(
+            array(
+'<?php
+    $e = new \\Exception(
+        \'test\',
+        100);
+    throw $e;',
+'<?php
+    $e = new \\Exception(\'test\',
+                         100);
+    throw $e;'),
```

Comment:
You open `(`, then start an element of array at next line, then you should put `)` on separate line just like with `(`.
And then - we just like to have commas after each element of multiline arrays ;)